src/Microsoft.Framework.FileSystemGlobbing/FilePatternMatch.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
```

Comment:
guh... It's such a pain. I wonder if we should have some kind of bot that can auto-add it :)